recipes/pipenv/meta.yaml

```
@@ -0,0 +1,56 @@
+  
+{% set name = "pipenv" %} 
+{% set version = "2018.11.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+
+# cuz source code didn't work, i used wheel instead.
+source:
+  url: "https://files.pythonhosted.org/packages/13/b4/3ffa55f77161cff9a5220f162670f7c5eb00df52e00939e203f601b0f579/pipenv-2018.11.26-py3-none-any.whl"
+  sha256: "56ad5f5cb48f1e58878e14525a6e3129d4306049cb76d2f6a3e95df0d5fc6330"
```

Quotes on the hash prevent the update bot from working
```suggestion
  sha256: 56ad5f5cb48f1e58878e14525a6e3129d4306049cb76d2f6a3e95df0d5fc6330
```